plugins/bundle/plugin.go

```diff
@@ -651,40 +655,6 @@ func (p *Plugin) configDelta(newConfig *Config) (map[string]*Source, map[string]
 	return newBundles, updatedBundles, deletedBundles
 }
 
-func (p *Plugin) saveBundleToDisk(name string, raw io.Reader) error {
```

Comment:
This function gives more details about what happened. So may be worth preserving this and updating `bundleUtils.SaveBundleToDisk` accordingly.

Reply:
Hmm, I looked into passing down the logger to SaveBundleToDisk and LoadBundleFromDisk but since there are no partial failure modes in the new implementations, and any error is logged by the caller in plugins/discovery/discovery.go and plugins/bundle/plugin.go it didn't seem worth adding another function param.

Wdyt? or have I missed the point and you were referring to something other than the in-function logging.